src/cli/mod.rs

```diff
@@ -354,9 +355,9 @@ impl ReplCli {
         backend.validate_field(field)?;
         match field {
             LanguageField::Pallas => repl!(rc, limit, pallas::Scalar, backend.clone()),
-            LanguageField::Vesta => todo!(),
-            LanguageField::BN256 => todo!(),
-            LanguageField::Grumpkin => todo!(),
+            LanguageField::Vesta => unimplemented!("Vesta field is not CurveCycleEquipped"),
+            LanguageField::BN256 => repl!(rc, limit, bn256::Fr, backend.clone()),
+            LanguageField::Grumpkin => unimplemented!("Grumpkin field is not CurveCycleEquipped"),
```

Comment:
TBH, I don't think any of Vesta or Grumpkin are ever going to make sense. What we're capturing here is the primary field, that is the one we use to define all the "payload" constraints.

*technically* Nova allows you to use any of the 2 sides of the curve cycle as primary interchangeably, but:
- that's no longer true with CycleFold,
- with pairing-based PCSes available on only one side of the curve, we'll want to put most constraints there.

It's probably worth asking @adr1anh for a double check of which field makes sense for the non-pairing cycles (looking up their moduli).

Reply:
IOW, I'm happy with what the PR has now, but we probably want to leave a TODO: "settle on one primary field only and remove the other"

Reply:
I left the undesired cases as `unreachable` as they should fail on the static check of field compatibility